inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/management/DynamicWorkloadManagementPage.html

```diff
@@ -17,84 +17,112 @@
  - limitations under the License.
 -->
 <html xmlns:wicket="http://wicket.apache.org" xmlns="http://www.w3.org/1999/html" lang="">
-<head>
 <wicket:head>
-  <style type="text/css">
-.page-content {
- overflow: auto;
-}
+    <style type="text/css">
+        .page-content {
+            overflow: auto;
+        }
 
-.table-striped>tbody>tr:nth-child(2n+1)>td, .table-striped>tbody>tr:nth-child(2n+1)>th
- {
- background-color: lightgray;
-}
+        .table-striped > tbody > tr:nth-child(2n+1) > td, .table-striped > tbody > tr:nth-child(2n+1) > th {
+            background-color: lightgray;
+        }
 
-.table-hover tbody tr:hover td, .table-hover tbody tr:hover th {
- background-color: burlywood;
-}
-</style>
+        .table-hover tbody tr:hover td, .table-hover tbody tr:hover th {
+            background-color: burlywood;
+        }
+
+        .form-control.k-widget {
+            display: inline-block;
+            height: 100px;
+            overflow-y: scroll;
+        }
+
+        .k-floatwrap {
+            height: 1000px;
+        }
+
+        .ddform {
```

Comment:
Is it really necessary to really extensively re-define this class locally here? If so, why?